client/components/data/query-wordads-status/index.jsx

```diff
@@ -36,16 +40,6 @@ class QueryWordadsStatus extends Component {
 	}
 }
 
-QueryWordadsStatus.propTypes = {
-	isRequestingWordadsStatus: PropTypes.bool,
-	requestWordadsStatus: PropTypes.func,
-	siteId: PropTypes.number,
-};
-
-QueryWordadsStatus.defaultProps = {
-	requestWordadsStatus: () => {},
-};
-
 export default connect(
 	( state, props ) => ( {
 		isRequestingWordadsStatus: isRequestingWordadsStatus( state, props.siteId ),
```

Comment:
The `isRequestingWordadsStatus` check was removed and this prop can be completely removed from the component, including the selector import and call. You only need the `requestWordadsStatus` action now.